docs/methods/na_screening.md

```diff
@@ -0,0 +1,126 @@
+Traffic light is determined based on:
+* type of contaminant
+* whether concentrations decrease over time or 
+* whether a relationship exists between BTEX and the redox conditions
+
+If the concentrations clearly decrease over time, or if the redox clearly changes over time and/or space, there is a chance that natural degradation is taking place (yellow traffic light). 
+If not, it must be decided that natural degradation is not a good choice (red traffic light). Benzene is very difficult to degrade under anaerobic conditions. So its behavior in relation to the other components must be monitored critically. 
+
+#### Overview
+
+Based on the development of the concentrations over time and/or the changes in redox in space and time, the color of the traffic light at the location can be determined as:
+
+| plume | redox and redox gradient and contamination in space and time      |    probability of NA    |
+| ------| ----------------------------------------------------------------- | ------------------------|
+| BTEX  | aerobic conditions | green |
+| BTEX  | anaerobic conditions, no redox gradient in space or decrease in concentrations in time | red |
+| BTEX  | anaerobic conditions, redox gradient in space: from anaerobic in the core zone to aerobic in the plume | yellow |
+| BTEX  | anaerobic conditions, concentrations decrease in time | yellow |
+| B     | anaerobic conditions, redox gradient in space and concentration gradient in time | yellow |
+| TEX   | anaerobic conditions, redox gradient in space and concentration gradient in time | green  |
+
+### Calculations
+
+Calculation of electron balance are based on the redox reactions, including stochiometric relations:
+
+Electrons are consumed through reduction. The reduction reactions for oxygen,
+nitrate, and sulfate are:
+
+* Oxygen: $4 e^- + 4 H^+ + 1 O_2 \rightarrow  H_2 O $
+* Nitrate: $ 5e^- + 6 H^+ + 1 NO_3^- \rightarrow 3 H_2O + 0.5 N_2 $
+* Sulfate: $ 8e^- + 9 H^+ 1 SO_4^{2-} \rightarrow 4H_2O + 1HS^{-} $
+
+Electrons are produced during oxidation of contaminants. Reactions included for selected, typically abundant contaminants are:
+
+* Benzene: $12 H_2O + 1 C_6H_6  \rightarrow 6CO_2 + 30 H^+ + 30e^- $
+* Toluene: $14 H_2O + 1 C_7H_8  \rightarrow 7CO_2 + 36 H^+ + 36e^- $
+* Ethylbenzene: $16 H_2O + 1 C_8H_{10}  \rightarrow 8CO_2 + 42 H^+ + 42e^- $
+* Xylene: $16 H_2O + 1 C_8H_{10}  \rightarrow 8CO_2 + 42 H^+ + 42e^- $
+* Indene: $18 H_2O + 1 C_9H_{8}  \rightarrow 9CO_2 + 44 H^+ + 44e^- $
+* Indane: $18 H_2O + 1 C_9H_{10}  \rightarrow 9CO_2 + 46 H^+ + 46e^- $
+* Naphtalene: $20 H_2O + 1 C_{10}H_{8}  \rightarrow 10CO_2 + 48 H^+ + 48e^- $
+
+Concentrations of the contaminants and redox conditions are divided by their molar mass to transform to molar concentrations.
+The total number of electrons is then calculated based on the stoichiometric relation for one liter of solution. Number of electrons 
+are added up for all reductors and oxidators per sample location.
+
+### Visualization
```

Comment:
Perhaps nice to add an example of the activity plot in this section. Can also be covered in a future PR.